pkg/virt-api/webhooks/validating-webhook/admitters/pod-eviction-admitter.go

```diff
@@ -36,12 +36,12 @@ func (admitter *PodEvictionAdmitter) Admit(ar *admissionv1.AdmissionReview) *adm
 
 	launcher := pod
 
-	domainName, exists := launcher.GetAnnotations()[virtv1.DomainAnnotation]
+	vmiName, exists := launcher.GetAnnotations()[virtv1.DomainAnnotation]
```

Comment:
This commit can belong to a bigger commit with the previous one (refactoring pod eviction admitted `Admit` function)

Reply:
Squashed it with the first commit.